package.json

```diff
@@ -98,6 +98,7 @@
     "rollup": "^2.47.0",
     "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-clear": "^2.0.7",
+    "rollup-plugin-copy": "^3.4.0",
```

Comment:
We also use `clean-css`, it's already available as a transitive dependency but we need to explicitly install it (in the same version as the curent one).